src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.card__stars {
+  display: flex;
+}
+
+.card__stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__stars-star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.card__stars--4 :nth-child(-n+4) {
```

Comment:
The BEM modifier should be named 'stars--4' instead of 'card__stars--4' to follow the proper BEM naming conventions.